modules/@ergo/core/store/dictionaries/actions.js

```diff
@@ -2,12 +2,13 @@
  * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
  * See LICENSE for license details.
  */
-import { modulesConfig } from '~/plugins/moduleLoader';
+import extendsModules from '~/.nuxt/extends.modules';
 import { types } from './mutations';
 
 export default {
     getDictionaries({ commit, rootState }) {
-        const { dictionaries: modulesDictionaries } = modulesConfig;
+        const modulesDictionaries = Object.values(extendsModules)
+            .reduce((acc, current) => [].concat(...acc, ...current.dictionaries || []), []);
```

Comment:
[].concat(...acc, ...current.dictionaries || []) might be done as [...acc, ...current.dictionaries]

Reply:
And instead of || [] condition, you might check if current.dictionaries exists then concate an arrays otherwise return acc. It might be faster - no need to recreate an array